src/corva/handlers.py

```diff
@@ -26,26 +26,31 @@ def wrapper(aws_event: Any, aws_context: Any) -> List[Any]:
                 handler=logging.StreamHandler(stream=sys.stdout),
                 logger=CORVA_LOGGER,
             ) as logging_ctx:
-                context = CorvaContext.from_aws(
-                    aws_event=aws_event, aws_context=aws_context
-                )
+                try:
+                    context = CorvaContext.from_aws(
+                        aws_event=aws_event, aws_context=aws_context
+                    )
 
-                api = Api(
-                    api_url=SETTINGS.API_ROOT_URL,
-                    data_api_url=SETTINGS.DATA_API_ROOT_URL,
-                    api_key=context.api_key,
-                    app_key=SETTINGS.APP_KEY,
-                    timeout=None,
-                )
+                    api = Api(
+                        api_url=SETTINGS.API_ROOT_URL,
+                        data_api_url=SETTINGS.DATA_API_ROOT_URL,
+                        api_key=context.api_key,
+                        app_key=SETTINGS.APP_KEY,
+                        timeout=None,
+                    )
+
+                    raw_events = raw_event_type.from_raw_event(event=aws_event)
 
-                raw_events = raw_event_type.from_raw_event(event=aws_event)
+                    results = [
+                        func(raw_event, api, context.aws_request_id, logging_ctx)
+                        for raw_event in raw_events
+                    ]
 
-                results = [
-                    func(raw_event, api, context.aws_request_id, logging_ctx)
-                    for raw_event in raw_events
-                ]
+                    return results
 
-                return results
+                except Exception:
+                    CORVA_LOGGER.exception('The app failed to execute.')
```

Comment:
Should we do this logging? Lambda will log unhandled exception itself, but without extra info like asset id and app connection id

Reply:
I think having the additional logging is good. We are always looking at issues by asset.